gen3/lib/testData/test1.manifest.g3k/expectedFenceResult.yaml

```diff
@@ -93,6 +95,27 @@ spec:
         # nginx group in current images
         fsGroup: 101
       containers:
+      - name: nginx-prometheus-exporter
```

Comment:
you need to have 3 sidecars to convert status endpoints into something prometheus can harvest?
can't we just have one script that does that, and writes out the `metrics.txt` in the fence container?

Reply:
I thought about using existing exporters instead of implementing our own.
But.. yeah, we could implement our own exporters later.